plugins/modules/azure_rm_datalakestore_info.py

```diff
@@ -0,0 +1,469 @@
+            type: str
+            sample: Succeeded
+        encryption_state:
+            description:
+                - The current state of encryption for this Data Lake Store account.
+            type: str
+            sample: Enabled
+        endpoint:
+            description:
+                - The full CName endpoint for this account.
+            returned: always
+            type: str
+            sample: testaccount.azuredatalakestore.net
+        firewall_allow_azure_ips:
+            description:
+                - The current state of allowing or disallowing IPs originating within Azure through the firewall.
+            type: str
+            sample: Disabled
+        firewall_rules:
+            description:
+                - The list of firewall rules associated with this Data Lake Store account.
+            type: list
+            contains:
+                name:
+                    description:
+                        - The resource name.
+                    type: str
+                    returned: always
+                    sample: Example Name
+                start_ip_address:
+                    description:
+                        - The start IP address for the firewall rule.
+                        - This can be either ipv4 or ipv6.
+                        - Start and End should be in the same protocol.
+                    type: str
+                    returned: always
+                    sample: 192.168.1.1
+                end_ip_address:
+                    description:
+                        - The end IP address for the firewall rule.
+                        - This can be either ipv4 or ipv6.
+                        - Start and End should be in the same protocol.
+                    type: str
+                    returned: always
+                    sample: 192.168.1.254
+        firewall_state:
+            description:
+                - The current state of the IP address firewall for this Data Lake Store account.
+            type: str
+            sample: Enabled
+        id:
+            description:
+                - The resource identifier.
+            returned: always
+            type: str
+            sample: /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.DataLakeStore/accounts/testaccount
+        identity:
+            description:
+                - The Key Vault encryption identity, if any.
+            type: complex
+            contains:
+                type:
+                    description:
+                        - The type of encryption being used.
+                    type: str
+                    sample: SystemAssigned
+                name:
+                    description:
+                        - The principal identifier associated with the encryption.
+                    type: str
+                    sample: 00000000-0000-0000-0000-000000000000
+                name:
```

Comment:
duplicate with 188 line.

Reply:
It was a mistake with copy and paste. I setted with correct name. Thanks!